applications/error-notifications.md

```diff
@@ -0,0 +1,32 @@
+# Error Notifications
+
+In order to alert teams to server errors for applications we need a way of
```

Comment:
Yes fine, I've not much experience of the errbit app to provide an eloquent description of its use and purpose